lua/projectlaunch/jobs.lua

```diff
@@ -5,7 +5,12 @@ local api = vim.api
 local term_name_prefix = "ProjectLaunch terminal - "
 
 local Job = {}
+runtimeVars = {}
```

Comment:
Could you do this slightly differently so `runtimeVars` doesn't need to be global? You could either move all the new functions to be part of `Job` like `Job:new` is, or move all of the functions inside of `generateCommand`.

My lua is a little rusty but I believe right now `runtimeVars` (and I think all the new functions) are all global, and it'd be really nice to not leak stuff to the global scope.